backand/controllers/facilities-controllers.js

```diff
@@ -0,0 +1,74 @@
+const HttpError = require('../models/error-http');
+
+
+let DUMMY_FACILITIES = [
+    {
+        Q1RQNVVeUldCUA: {
+            national: false,
+            updated: "2016-06-29T12:42:22",
+            postal_code: "94612",
+            location_type_id: 1,
+            municipal: true,
+            city: "Oakland",
+            event_only: false,
+            latitude: 37.80604715391224,
+            province: "CA",
+            fax: "",
+            description: "Elihu Harris State Building",
+            curbside: false,
+            hours: "Monday to Friday 8am to 5pm",
+            phone: "(510) 287-1651",
+            address: "1515 Clay Street",
+            notes_public: "Please look for the East Bay Municipal Utility District (EBMUD) bin. EBMUD provides free pharmaceutical disposal bins at nine East Bay locations to keep them out of the Bay.\r\n\r\nPlease take pills out of their bottles and put in a plastic zipper bag. For liquids, please block out your personal information on the container and put in a leak-proof plastic bag. Drop the bagged medicines into the disposal bin.",
+            created: "2012-05-10T16:38:49",
+            url: "https://www.ebmud.com/wastewater/bay-friendly-waste-disposal/medicine-disposal/",
+            country: "US",
+            region: "Alameda",
+            longitude: -122.27331877704263,
+            geocoded: false,
+            materials: [
+                {
+                    dropoff: true,
+                    description: "Medications",
+                    business: false,
+                    url: "",
+                    residential: true,
+                    notes: "This site accepts prescription drugs, over-the-counter medications, pet medicines and prescription strength liquids and creams.",
+                    residential_method: "dropoff",
+                    business_method: "none",
+                    material_id: 193,
+                    pickup: false,
+                    pending: "F"
+                }
+            ],
+            notes: "Please look for the East Bay Municipal Utility District (EBMUD) bin. EBMUD provides free pharmaceutical disposal bins at nine East Bay locations to keep them out of the Bay.\r\n\r\nPlease take pills out of their bottles and put in a plastic zipper bag. For liquids, please block out your personal information on the container and put in a leak-proof plastic bag. Drop the bagged medicines into the disposal bin."
+    }},
+];
+
+
+const getAllFacilities = (req, res, next) => {
+    console.log('GET in facility-routes');
+
+    // If there is no facility found, NOT FOUND 404 will send back
+    // But, this is odd???
```

Comment:
Why is this odd? This is a dummy method right? There will always be facilities? Or will dummy-facilities be replaced with state var later?

Reply:
Yeah, it is because DUMMY data is always available at this point. The error checking itself at this stage looks odd. I should remove the confusing comment. Eventually, this error checking will become more reasonable because it is possible to get no facility.